@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -393,6 +399,8 @@ export class VisibleTileSet {
         private readonly m_tileGeometryManager: TileGeometryManager,
         public options: VisibleTileSetOptions
     ) {
+        this.options = options;
+        this.options.maxTilesPerFrame = Math.floor(this.options.maxTilesPerFrame ?? 100);
```

Comment:
Instead of having an arbitrary number, I would rather use 0 by default and treat 0 as unlimited amount of tiles

Reply:
done